provider/property_override.rb

```diff
@@ -20,6 +20,16 @@ module Provider
   class PropertyOverride < Api::Object
     attr_reader :new_type
 
+    # Ordering of imports can be important so this property can make the
```

Comment:
I don't quite know what you mean by "ordering of imports".

Reply:
Oh I meant to say "ordering of overrides". Such that if 1 override is applied before a different one (in this case if Image is applied before SourceImage) then MM will break. This is because Source image is adding properties that the Image override depends on.